lib/libuutil/libuutil.abi

```diff
@@ -616,6 +616,7 @@
     <array-type-def dimensions='1' type-id='de572c22' size-in-bits='1472' id='6d3c2f42'>
       <subrange length='23' type-id='7359adad' id='fdd0f594'/>
     </array-type-def>
+    <type-decl name='long long int' size-in-bits='64' id='1eb56b1e'/>
```

Comment:
There shouldn't be any ABI changes after the switch to `zhack` so we can drop the .abi file updates.